src/main/java/com/synopsys/integration/detect/lifecycle/run/RunManager.java

```diff
@@ -445,6 +468,52 @@ private void runBlackDuckProduct(ProductRunData productRunData, DetectConfigurat
         }
     }
 
+    // When running the signature scanner on a non-docker target
+    private void runSignatureScanner(DetectConfigurationFactory detectConfigurationFactory, BlackDuckRunData blackDuckRunData, EventSystem eventSystem, NameVersion projectNameVersion, CodeLocationAccumulator codeLocationAccumulator)
+        throws DetectUserFriendlyException {
+        runSignatureScanner(detectConfigurationFactory, blackDuckRunData, DockerTargetData.NO_DOCKER_TARGET, eventSystem, projectNameVersion, codeLocationAccumulator);
```

Comment:
It seems simpler to me to just to call it with DockerTargetOptional.orElse(NO_DOCKER_TARGET) instead of wrapping the method.